src/main/java/slimeknights/tconstruct/common/data/TConstructRecipeProvider.java

```diff
@@ -598,6 +599,33 @@ private void addSmelteryRecipes(Consumer<IFinishedRecipe> consumer) {
     this.registerSlabStair(consumer, TinkerSmeltery.searedCreeper, folder, true);
     this.registerSlabStair(consumer, TinkerSmeltery.searedRoad, folder, true);
     this.registerSlabStair(consumer, TinkerSmeltery.searedTile, folder, true);
+
+    ShapedRecipeBuilder.shapedRecipe(TinkerSmeltery.searedTank.get(SearedTankBlock.TankType.TANK))
+      .key('#', TinkerSmeltery.searedBrick)
+      .key('B', Tags.Items.GLASS)
+      .patternLine("###")
+      .patternLine("#B#")
+      .patternLine("###")
+      .addCriterion("has_item", this.hasItem(TinkerSmeltery.searedBrick))
+      .build(consumer, "tconstruct:smeltery/seared/tank");
```

Comment:
Use `location`, keeps us from hardcoding mod and makes the code a bit cleaner looking.

Reply:
Not sure what you'd have me put here. I get the idea, but I don't quite get all the things.

Reply:
I think he meant ResourceLocation or a function named `location` that returns one

Reply:
There is a function in this case named `location`, right? I remember putting it there.